receiver/oracledbreceiver/metadata.yaml

```
@@ -0,0 +1,176 @@
+attributes:
+  db.oracle.session.status:
+    description: Session status
+    type: string
+  db.oracle.session.type:
+    description: Session type
+    type: string
+  db.oracle.tablespace.name:
+    description: Tablespace name
```

Forgot to comment on this one. We don't specify such long attributes on datapoint level because each metric already has that in its name. So I think these should be
```suggestion
  session_status:
    description: Session status
    type: string
  session_type:
    description: Session type
    type: string
  tablespace_name:
    description: Tablespace name
```

---

addressed